src/Core/Manifest/ClassManifest.php

```diff
@@ -382,15 +384,6 @@ public function getOwnerModule($class)
     }
 
     /**
-     * Used to set up files that we want to exclude from parsing for performance reasons.
```

Comment:
Damian confirmed that this is no longer the case (memory usage on flush doesn't go up). I'm unsure why that would be, since we haven't really fixed the underlying issue (tokenizer running over a class with heaps of cyclomatic complexity). It might be that the tokenizer in PHP 5.6 is more memory efficient.